classes/QuickAccess.php

```diff
@@ -125,7 +125,7 @@ public static function getQuickAccessesWithToken($idLang, $idEmployee)
      */
     public function toggleNewWindow()
     {
-        if (!array_key_exists('new_window', $this)) {
+        if (!array_key_exists('new_window', get_object_vars($this))) {
```

Comment:
Why not use `property_exists` ?

Reply:
Because property_exists returns true when the property is private or protected

Reply:
Oh !